lib/stimulus_reflex/reflex.rb

```diff
@@ -61,6 +56,10 @@ def reflex_id
   end
   # END TODO: remove
 
+  def logger
+    StimulusReflex::Logger.new(self) unless suppress_logging
```

Comment:
Isn't it preferable to keep the logger instance memoized on the reflex like it was before, instead of re-initializing it on each log call? Is there a specific issue this change is addressing?

Reply:
Thanks, I have added memoization. The reason for moving it out of `#initialize` is complexity reduction.